### ISSUE TITLE
ENOENT on an unexisting partial

### ISSUE THREAD

**Issue by faddee** [tgreyuk/typedoc-plugin-markdown#4]
You're quick. Almost there. :-)

```
Error: ENOENT: no such file or directory, open 'C:\path\to\node_modules\typedoc-plugin-markdown\dist\theme\partials\member.objectliteral.hbs'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.readFileSync (fs.js:508:33)
    at templateToString (C:\path\to\node_modules\typedoc-plugin-markdown\dist\theme\utils.js:13:15)
    at Object.compileTemplate (C:\path\to\node_modules\typedoc-plugin-markdown\dist\theme\utils.js:7:18)
    at DeclarationReflection.compileMember (C:\path\to\node_modules\typedoc-plugin-markdown\dist\theme\helpers\compileMember.js:15:26)
    at eval (eval at createFunctionContext (C:\path\to\node_modules\handlebars\dist\cjs\handlebars\compiler\javascript-compiler.js:254:23), <anonymous>:5:106)
    at Object.prog [as inverse] (C:\path\to\node_modules\handlebars\dist\cjs\handlebars\runtime.js:221:12)
    at DeclarationReflection.<anonymous> (C:\path\to\node_modules\handlebars\dist\cjs\handlebars\helpers\if.js:17:22)
    at DeclarationReflection.<anonymous> (C:\path\to\node_modules\handlebars\dist\cjs\handlebars\helpers\if.js:24:35)
```

**Comment by tgreyuk** [comment]
I think thats related to a case sensitivity issue as the file should be there. Anyway have refactored that file out now so should be fixed in 0.0.13.

**Comment by faddee** [comment]
Works great now. Thanks!
